shell/common/fixtures/shell_test.dart

```diff
@@ -629,3 +630,13 @@ void renderWarmUpView1and2() {
     }
   });
 }
+
```

Comment:
I wired up a test to verify this behavior but found that it was passing anyway. I don't know if this is due to the test environment, but 🤷‍♂️